src/main/java/mcscheduler/logic/commands/TakeLeaveCommand.java

```diff
@@ -31,7 +31,7 @@
             + "during the specified shift by the index numbers used in the last worker and shift listings. "
             + "\nParameters: "
             + PREFIX_SHIFT + "SHIFT_INDEX (must be a positive integer) "
-            + "[" + PREFIX_WORKER + "WORKER_INDEX (must be a positive integer)]...\n"
+            + PREFIX_WORKER + "WORKER_INDEX (must be a positive integer)...\n"
```

Comment:
same as `cancel-leave`, should still have square brackets as can take multiple workers?
```suggestion
            + "[" + PREFIX_WORKER + "WORKER_INDEX (must be a positive integer)]...\n"
```

Reply:
Same as my reply above as well